arangod/RocksDBEngine/RocksDBZkdIndex.cpp

```diff
@@ -219,12 +219,24 @@ class RocksDBZkdIndexIterator final : public IndexIterator {
   bool nextCoveringImpl(CoveringCallback const& callback,
                         uint64_t limit) override {
     struct CoveringData : IndexIteratorCoveringData {
-      explicit CoveringData(velocypack::Slice data) : _data(data) {}
-      VPackSlice at(size_t i) const override { return _data.at(i); }
+      CoveringData(VPackSlice prefixValues, VPackSlice storedValues)
```

Comment:
is there a reason that this constructor is not marked explicit anymore?

Reply:
yes, it has now two parameters. They can't be used in implicit conversions, right?